modAion/src/org/aion/zero/exceptions/HeaderStructureException.java

```diff
@@ -56,7 +56,7 @@ public HeaderStructureException(String field, int offset) {
         this.offset = offset;
     }
 
-    public HeaderStructureException(String field, int offset, Throwable cause) {
+    public HeaderStructureException(String field, int offset, Exception cause) {
```

Comment:
Throwable should not be caught, for the purpose of avoiding catching Errors that generally can't be handled gracefully.  However, I think it is fine for an Exception subclass to take a Throwable as its cause.  For instance, the class java.lang.Exception allows cause to be a Throwable.  I think we should follow that pattern/design.